test/util/test_runners/integrations_tests_runner.py

```diff
@@ -0,0 +1,136 @@
+"""
+This is a runner module for all integration tests.
+The purpose of this file is to be a cross platform tests runner
+for all integration tests
+
+Roadmap:
+1) run all integration unitests from this file
+2) set up the system env to run the unitests
+"""
+
+# biultIn
+import sys
+import os
+
+# Third party
+import unittest
+import base64
+import requests
+from pathlib import Path
+import logging
+import subprocess as sb
+import shutil
+
+# internals
+from test.test_cli_integration import CliIntegrationTest, CliIntegrationTestConfigurations
+from conjur.endpoints import ConjurEndpoint
+from test.util.test_runners.integration_test_case import IntegratioTestCaseBase
+from test.util.test_runners.test_runner_args import TestRunnerArgs
+from test.util.test_runners.params import ClientParams, TestEnvironmentParams
+from test.util.test_path_provider import TestRunnerPathProvider
+
+
+class TestRunner:
+
```

Comment:
missing doc string here

Reply:
Wasn't sure what to put in it since it all specifies in the module docs..
What is the reason we docString everything? is it for the IDE hints?

Reply:
we use a pylinter that screams at us. 
Here specifically I think it is important to understand where the entrypoint is of the code